pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -240,6 +240,15 @@ type GracefulEvictionTask struct {
 	// +required
 	FromCluster string `json:"fromCluster"`
 
+	// PurgeMode represents how to deal with the legacy applications on the
+	// cluster from which the application is migrated.
+	// Valid options are "Immediately", "Graciously" and "Never".
+	// Defaults to "Graciously".
+	// +kubebuilder:validation:Enum=Immediately;Graciously;Never
+	// +kubebuilder:default=Graciously
```

Comment:
For application failover, the PurgeMode should be inherited from PropagationPolicy which manages the current workload, but for cluster failover, as I mentioned at #5788, we can set `Graciously` as the default.

So, from the API perspective, it might not have the default preference. 

```suggestion
	// +kubebuilder:validation:Enum=Immediately;Graciously;Never
```

Reply:
Does it need to be a required field?

Reply:
Since we inherit PurgeMode from the PropagationPolicy (which has a default) and will have a default value for cluster failover, I don't believe this field should ever be empty. I can change this to required if we'd like.

Reply:
It's not necessarily necessary, it's just a question of mine.